templates/web/base/develop/email_list.html

```diff
@@ -0,0 +1,11 @@
+[% INCLUDE 'header.html' %]
+
+<h1>[% loc('Templates') %]</h1>
+
+<ul>
+[% FOR template IN templates %]
+<li><a href="[% template.url %]">[% template.name %]</a></li>
```

Comment:
Relative `href` here (eg: `href="alert-confirm"`) works if you access the email dev page via the URL `http://fixmystreet.local:3000/_dev/email/` (with a trailing slash) but _not_ if you access it via `http://fixmystreet.local:3000/_dev/email` (without the trailing slash).

Just mention it because, at first, none of the links worked for me, and I realised it was because I was missing a trailing slash, so all the relative `href`s were leading to places like `http://fixmystreet.local:3000/_dev/alert-confirm`. I wonder whether there’s a simple way to avoid that?

Reply:
Hmm, yes, will look at that.

Reply:
Okay, it should work at either now.

Reply:
:+1: